frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/advanced-tab/advanced-tab.scss

```diff
@@ -0,0 +1,8 @@
+.AdvancedTab-tab-title {
+  .pf-c-accordion__toggle-text {
+    display: flex;
+    .pf-c-form__group-label-help {
+      margin-left: 6px;
```

Comment:
we should not use exact numbers in CSS.
try `var(--pf-global--spacer--md)` or `var(--pf-global--spacer--lg)`